tests/mypy/outputs/1.0.1/mypy-plugin_ini/root_models.py

```diff
@@ -16,8 +16,10 @@ class Pets3(RootModel):
 
 
 pets1 = Pets1(['dog', 'cat'])
+# MYPY: error: Too many positional arguments for "Pets1"  [misc]
 pets2 = Pets2(['dog', 'cat'])
 pets3 = Pets3(['dog', 'cat'])
+# MYPY: error: Too many positional arguments for "Pets3"  [misc]
```

Comment:
mypy support for dataclass_transform isn't really good iirc, hence the errors

Reply:
@Viicos, can't we remove these now that you added back support for 1.0.1?